docker-wrappers/OmicsIntegrator2/Dockerfile

```diff
@@ -2,5 +2,14 @@
 # https://github.com/fraenkel-lab/OmicsIntegrator2
 FROM continuumio/miniconda3:4.9.2
 
+COPY 0001-disable-graph-exports.patch .
```

Comment:
We'll have to make the usual updates elsewhere in SPRAS that accompany a Docker image bump: image version, note in the readme, etc.

Reply:
(This can be made easier with #314!)

Reply:
I don't know how soon we'll get to #314 so we may need to make these related changes manually to get this merged.